srcpkgs/ffms2/template

```diff
@@ -1,19 +1,21 @@
 # Template file for 'ffms2'
 pkgname=ffms2
-version=2.23.1
-revision=3
-_version=2.23
-wrksrc="${pkgname}-${_version}"
+version=2.40
+revision=1
 build_style=gnu-configure
 configure_args="--enable-shared --disable-static"
-hostmakedepends="pkg-config"
+hostmakedepends="automake libtool pkg-config"
 makedepends="ffmpeg-devel zlib-devel"
-short_desc="A libav/ffmpeg based frames server"
+short_desc="Cross-platform wrapper library around FFmpeg"
 maintainer="DirectorX <void.directorx@protonmail.com>"
 license="MIT, GPL-3.0-or-later"
 homepage="https://github.com/FFMS/ffms2"
-distfiles="${homepage}/archive/${_version}.tar.gz"
-checksum=b09b2aa2b1c6f87f94a0a0dd8284b3c791cbe77f0f3df57af99ddebcd15273ed
+distfiles="https://github.com/FFMS/ffms2/archive/${version}.tar.gz"
+checksum=82e95662946f3d6e1b529eadbd72bed196adfbc41368b2d50493efce6e716320
+
+pre_configure() {
+	./autogen.sh
```

Comment:
try to run only `autoreconf -fi

Did they stop distributing a pre genereated `configure`?

Reply:
I suppose.